org.kde.digikam.yml

```diff
@@ -0,0 +1,243 @@
+          - -DWITH_IMGCODEC_SUNRASTER=OFF
+          - -DWITH_IMGCODEC_PXM=OFF
+          - -DWITH_CUDA=OFF
+          - -DWITH_CUFFT=OFF
+          - -DWITH_CUBLAS=OFF
+          - -DWITH_NVCUVID=OFF
+          - -DWITH_OPENCL=OFF
+          - -DWITH_OPENCL_SVM=OFF
+          - -DWITH_OPENCLAMDFFT=OFF
+          - -DWITH_OPENCLAMDBLAS=OFF
+          - -DCV_ENABLE_INTRINSICS=OFF
+          - -DCV_DISABLE_OPTIMIZATION=OFF
+          - -DCV_TRACE=OFF
+          - -DWITH_OPENJPEG=OFF
+        cleanup:
+          - /bin
+        sources:
+          - type: git
+            url: https://github.com/Itseez/opencv.git
+            tag: 4.3.0
+            commit: 01b2c5a77ca6dbef3baef24ebc0a5984579231d9
+      - name: exiv2
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DBUILD_SHARED_LIBS=ON
+          - -DEXIV2_ENABLE_XMP=ON
+          - -DEXIV2_ENABLE_EXTERNAL_XMP=OFF
+          - -DEXIV2_ENABLE_PNG=ON
+          - -DEXIV2_ENABLE_HEIF=ON
+          - -DEXIV2_ENABLE_NLS=ON
+          - -DEXIV2_ENABLE_PRINTUCS2=ON
+          - -DEXIV2_ENABLE_LENSDATA=ON
+          - -DEXIV2_ENABLE_VIDEO=OFF
+          - -DEXIV2_ENABLE_WEBREADY=OFF
+          - -DEXIV2_ENABLE_DYNAMIC_RUNTIME=OFF
+          - -DEXIV2_ENABLE_CURL=OFF
+          - -DEXIV2_ENABLE_SSH=OFF
+          - -DEXIV2_BUILD_SAMPLES=OFF
+          - -DEXIV2_BUILD_PO=OFF
+          - -DEXIV2_BUILD_EXIV2_COMMAND=OFF
+          - -DEXIV2_BUILD_UNIT_TESTS=OFF
+          - -DEXIV2_BUILD_DOC=OFF
+          - -DEXIV2_TEAM_EXTRA_WARNINGS=OFF
+          - -DEXIV2_TEAM_WARNINGS_AS_ERRORS=OFF
+          - -DEXIV2_TEAM_USE_SANITIZERS=OFF
+          - -DEXIV2_ENABLE_WIN_UNICODE=OFF
+        sources:
+          - type: git
+            url: https://github.com/Exiv2/exiv2.git
+            tag: v0.27.2
+            commit: 70f9405fafbe19c7bab94b36a1267c4332dc8a8a
+      - name: marble
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DWITH_DESIGNER_PLUGIN=OFF
+          - -DBUILD_MARBLE_TESTS=OFF
+          - -DBUILD_MARBLE_TOOLS=OFF
+          - -DBUILD_MARBLE_EXAMPLES=OFF
+          - -DBUILD_MARBLE_APPS=OFF
+          - -DBUILD_WITH_DBUS=ON
+          - -DBUILD_TESTING=OFF
+        cleanup:
+          - /mkspecs
+        sources:
+          - type: git
+            url: https://invent.kde.org/education/marble.git
+            tag: v20.04.1
+            commit: e3483bec0a7e79b7685055512712f79efaf19deb
+      - name: lensfun
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+          - -DBUILD_STATIC=OFF
+          - -DBUILD_WITH_MSVC_STATIC_RUNTIME=OFF
+          - -DBUILD_TESTS=OFF
+          - -DBUILD_LENSTOOL=OFF
+          - -DBUILD_DOC=OFF
+          - -DINSTALL_PYTHON_MODULE=OFF
+          - -DINSTALL_HELPER_SCRIPTS=OFF
+        sources:
+          - type: git
+            url: https://github.com/lensfun/lensfun.git
+            tag: v0.3.95
```

Comment:
Don't use that version, see:

https://github.com/flathub/org.darktable.Darktable/pull/59

Reply:
No. Please use Lensfun from git master it work perfectly with digiKam. Using older 0.3.2 will introduce huge regressions has lens database is very outdated...

Reply:
Why I should not use v0.3.95? In my photo collection all works fine, but maybe I didn't test all stuff. How I can check it?

Reply:
Please read the PR, it updates the database independently from the library.

Reply:
If @cgilles as the upstream maintainer is ok with it then go. I just hope this will not lead to incompatibilities with distribution packages of digikam, that are built against 0.3.x.